nornir_jinja2/plugins/tasks/template_string.py

```diff
@@ -31,6 +31,7 @@ def template_string(
 
     if jinja_env:
         env = jinja_env
+        env.loader = FileSystemLoader(path)
```

Comment:
this one doesn't accept path as the template is passed directly as a string

Reply:
Ah, of course. Sorry I didn't even check